HipoProject/urls.py

```diff
@@ -26,9 +26,9 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/profiles/create/', UserRegistrationView.as_view(), name='create_user'),
-    path('api/profiles/authenticate/', UserAuthenticationView.as_view()),
-    path('api/profiles/me/', MyProfileDetailView.as_view()),
-    path('api/profiles/me/change-password/', ChangePasswordView.as_view()),
+    path('api/profiles/authenticate/', UserAuthenticationView.as_view(), name='authenticate_user'),
+    path('api/profiles/me/', MyProfileDetailView.as_view(), name='profile_detail'),
+    path('api/profiles/me/change-password/', ChangePasswordView.as_view(), name='change_password'),
```

Comment:
Generally conversion is using dash (-) for naming views. 
Ex. `change-password`
https://docs.djangoproject.com/en/2.2/topics/http/urls/#examples

We can use `my-profile-detail`. What do you think ?